locale/pt-br/texts.po

```diff
@@ -0,0 +1,1422 @@
+msgid ""
+msgstr ""
+"Language: pt-br\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals = 2; plural = (n != 1);\n"
+
+#: src/constants.js:44
+msgid "I want to reset my wallet"
+msgstr "Eu quero resetar minha wallet"
+
+#: src/screens/ChoosePassphrase.js:48
+msgid "To set a blank passphrase mark the corresponding checkbox above."
+msgstr "Marque acima para deixar a frase secreta em branco."
+
+#: src/screens/ChoosePassphrase.js:54
+msgid "Passphrase and confirm passphrase must be equal"
+msgstr "A frase secreta e sua confirmação devem ser iguais"
+
+#: src/components/ModalBackupWords.js:99 src/components/ModalResetAllData.js:41
+#: src/screens/ChoosePassphrase.js:59
+msgid "Invalid password"
+msgstr "Senha inválida"
+
+#: src/components/ModalPin.js:57 src/screens/ChoosePassphrase.js:64
+#: src/screens/LockedWallet.js:52
+msgid "Invalid PIN"
+msgstr "PIN inválido"
+
+#: src/screens/ChoosePassphrase.js:105
+msgid ""
+"Are you sure you want to change your whole wallet setting this passphrase?"
+msgstr "Você tem certeza que deseja definir uma frase secreta?"
+
+#: src/screens/ChoosePassphrase.js:115
+msgid "I understand the risks of adding a passphrase."
+msgstr "Eu entendo os riscos de adicionar uma frase secreta."
+
+#: src/screens/ChoosePassphrase.js:132
+msgid "I want to set a blank passphrase."
+msgstr "Eu quero usar uma frase secreta vazia."
+
+#: src/screens/ChoosePassphrase.js:136
+msgid "Please, enter your password and PIN to confirm the operation."
+msgstr "Por favor, entre sua senha e seu PIN para confirmar a operação"
```

Comment:
I think I prefer 'digite' as the translation of 'enter' in this context. What do you think?

Reply:
Fixed. Thanks!